runtime/doc/treesitter.txt

```diff
@@ -880,8 +878,8 @@ start({bufnr}, {lang})                                *vim.treesitter.start()*
     Can be used in an ftplugin or FileType autocommand.
 
     Note: By default, disables regex syntax highlighting, which may be
-    required for some plugins. In this case, add `vim.bo.syntax = 'on'` after
-    the call to `start`.
+    required for some plugins. In this case, add ``vim.bo.syntax = 'on'``
```

Comment:
👀

Reply:
This is what it is in `treesitter.lua`. My new scripts have taken it verbatim.